dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -290,6 +290,8 @@ app.on('ready', () => {
     // Check whether we are in bundled application or development mode.
     // @ts-ignore
     const devMode = process.defaultApp || /node_modules[\/]electron[\/]/.test(process.execPath);
+    // Check if user wants to run everything as one process.
+    const noBackendFork = Boolean(process.env['THEIA_ELECTRON_NO_BACKEND_FORK']) || process.argv.includes('--no-cluster');
```

Comment:
I'm not sure about adding `process.env['THEIA_ELECTRON_NO_BACKEND_FORK']`. I'm afraid to open up the approach which potentially breaks performance.

@thegecko Will you use it to run the server process inside the electron main?